src/consumption.cpp

```diff
@@ -556,6 +558,9 @@ bool player::eat( item &food, bool force )
         if( has_trait( trait_id( "MOUTH_TENTACLES" ) )  || has_trait( trait_id( "MANDIBLES" ) ) ||
             has_trait( trait_id( "FANGS_SPIDER" ) ) ) {
             mealtime /= 2;
+        } else if( has_trait( trait_id( "SHARKTEETH" ) ) ) {
+            //SHARKBAIT! HOO HA HA!
+            mealtime = ( int )( mealtime / 3 );
```

Comment:
Why are you casting (using a C-style cast none the less) an `int` to an `int`? (Hint: `mealtime` is `int`, so is `3`, and therefor so is `mealtime / 3`.)

Reply:
In the past I've had issues because `mealtime` is an even number and dividing even numbers by an odd one has thrown errors. Does C++ handle rounding automatically?

Reply:
> In the past I've had issues because mealtime is an even number and dividing even numbers by an odd one has thrown errors. Does C++ handle rounding automatically?

There a re plenty of sites explaining the behaviour of arithmetic operators, e.g. https://en.cppreference.com/w/cpp/language/operator_arithmetic - read about this instead of guessing.

In short: integer division discards the remainder, e.g. `5 / 3 == 1`. If either operand is of a floating point type and the other isn't, both are converted to floating point and division is as you would expect it.